frontend/src/__tests__/cypress/cypress/pages/home/homeAIFlow.ts

```diff
@@ -0,0 +1,41 @@
+import { Contextual } from '~/__tests__/cypress/cypress/pages/components/Contextual';
+
+class AIFlowCard extends Contextual<HTMLElement> {}
+
+export class HomeAIFlow extends Contextual<HTMLElement> {
+  getProjectCard(): AIFlowCard {
+    return new AIFlowCard(() => cy.findByTestId('ai-flow-projects-card'));
+  }
+
+  getTrainFlowCard(): AIFlowCard {
+    return new AIFlowCard(() => cy.findByTestId('ai-flow-train-card'));
+  }
+
+  getModelsFlowCard(): AIFlowCard {
+    return new AIFlowCard(() => cy.findByTestId('ai-flow-models-card'));
+  }
+
+  findAIFlowInfo(infoName: string): Cypress.Chainable<JQuery<HTMLElement>> {
+    return cy.findByTestId(`ai-flows-${infoName}-info`);
+  }
+
+  clickAIFlowClose(): void {
+    cy.findByTestId('ai-flows-close-info').click();
+  }
```

Comment:
Should return the selector:
```suggestion
  findAIFlowClose(): void {
    return cy.findByTestId('ai-flows-close-info');
  }
```